### PR TITLE
Refs [Task][82598] Author information

### PR DESCRIPTION
**Pull request by vunt-3611**
## Related Tickets
- [#82598](https://edu-redmine.sun-asterisk.vn/issues/82598)

## WHAT (optional)
- Làm màn thông tin chi tiết tác giả

## Evidence (Screenshot or Video)
Rubocop:
image

### CODE REVIEW

**Comment by vunt-3611** [comment]
ready

File: app/views/authors/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    <div class="md:w-2/3 p-6">
+      <div class="mb-6">
+        <h2 class="text-xl font-semibold text-gray-700 border-b pb-2 mb-4">
+          <%= t("view.author.book_in_system") + ": " + @author.books.count.to_s %>
+        </h2>
+      </div>
+      <div class="mb-6">
+        <h2 class="text-xl font-semibold text-gray-700 border-b pb-2 mb-4">
+          <%= t("view.author.biography")%>
```
Comment (ghost) [inline app/views/authors/show.html.erb]:
```suggestion
          <%= t("view.author.biography") %>
```

File: app/views/authors/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        </h2>
+        <p class="text-gray-600 leading-relaxed">
+          <%= biography @author %>
+        </p>
+      </div>
+
+      <div>
+        <h2 class="text-xl font-semibold text-gray-700 border-b pb-2 mb-4">
+          <%= t("view.author.written_books")%>
```
Comment (ghost) [inline app/views/authors/show.html.erb]:
```suggestion
          <%= t("view.author.written_books") %>
```

File: app/views/authors/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </h2>
+        <% if @written_books.any? %>
+          <div class="grid md:grid-cols-2 gap-4 max-h-[370px] overflow-y-auto">
+            <%= render partial: "written_book", collection: @written_books, as: :book %>
+          </div>
+        <% else %>
+          <p class="text-gray-500 text-center py-4">
+            <%= t("view.author.no_book")%>
```
Comment (ghost) [inline app/views/authors/show.html.erb]:
```suggestion
            <%= t("view.author.no_book") %>
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by vunt-3611** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**